src/liblsquic/lsquic_crypto.c

```diff
@@ -281,14 +284,14 @@ void lshkdf_extract(const unsigned char *ikm, int ikm_len, const unsigned char *
 }
 
 
+#define SHA256LEN   32
 int lshkdf_expand(const unsigned char *prk, const unsigned char *info, int info_len,
                 uint16_t c_key_len, uint8_t *c_key,
                 uint16_t s_key_len, uint8_t *s_key,
                 uint16_t c_key_iv_len, uint8_t *c_key_iv,
                 uint16_t s_key_iv_len, uint8_t *s_key_iv,
                 uint16_t sub_key_len, uint8_t *sub_key)
 {
-    const int SHA256LEN = 32;
```

Comment:
This is interesting: why does MSVC complain about this?

Reply:
It doesn't like even consts as array dimensions.  It's a really half-assed C99 support.